milton-processor/scribe.ts

```diff
@@ -40,14 +40,15 @@ app.options("/simplify", (req, res) => {
   // Allow CORS requests from CDN
   res.set('Access-Control-Allow-Origin', CONFIG.cors_client);
   res.set('Access-Control-Allow-Methods', 'POST');
-  res.set('Access-Control-Allow-Headers', 'Content-Type');
+  res.set('Access-Control-Allow-Headers', ['Content-Type', 'token']);
```

Comment:
Might be better to use the standard Authentication header (with Bearer scheme)? I don't think it's critical, but it might have some implications for CORS.

Reply:
True, although t's been working with my frontend via CORS, so I'm gonna leave it for now.